dev_guide/builds.adoc

```diff
@@ -629,7 +629,9 @@ located at `_<system-registry>_/_<namespace>_/ruby-20-centos7`. The
 `_<system-registry>_` is defined as a service with the name `docker-registry`
 running in OpenShift.
 
-. Define a build with a strategy that consumes the image stream:
+. Define a build with a trigger that points to the image stream. If the
```

Comment:
?  not sure why this was wrong as it was?  we're not up to the point of defining triggers yet, just the strategy From